syntax = "proto3";

service RPC {
  rpc CountCrowd (CountCrowdRequest) returns (CountCrowdReply) {}
}

message CountCrowdRequest {
  string version = 1;
  bytes image = 2;
}

message CountCrowdReply {
  string version = 1;
  bytes density_map = 2;
  float crowd_count = 3;
  float line_count = 4;
}
